src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function sortListSalary(ul) {
+  const items = [...list.querySelectorAll('li')];
```

Comment:
Do not use querySelectorAll in this function. It should recieve this list as argument